images/mediatypes.go

```diff
@@ -22,8 +22,10 @@ package images
 // here for clarity.
 const (
 	MediaTypeDockerSchema2Layer            = "application/vnd.docker.image.rootfs.diff.tar"
+	MediaTypeDockerSchema2LayerEnc         = "application/vnd.docker.image.rootfs.diff.tar+enc"
```

Comment:
Do the media types need to specify the encryption type?

Reply:
Nope, the way we specify them are with this scheme: 
https://github.com/lumjjb/image-spec/blob/encryption/layer.md#layer-encryption